### ISSUE TITLE
Text view rendered wrongly when using bitmap font

### ISSUE THREAD

**Issue by jobe-m** [korlibs/korge#2074]
When using a bitmap font with `Text()` then the text will be rendered missplaced (it looks like 180° rotated).

![image](https://github.com/korlibs/korge/assets/19785115/3494ba47-363c-41e3-baf5-f74e4306f975)

The code to reproduce this:

```
    text("Text") {
        x = 100.0
        y = 150.0
        smoothing = false
        font = resourcesVfs["m5x7_16_outline.fnt"].readBitmapFont()
    }
```

When using the default font (remove line `font = ...`) then the text will be rendered correctly.

Here are the .fnt and .png files: (remove .txt extention, GitHub needs it for upload)

[m5x7_16_outline.fnt.txt](https://github.com/korlibs/korge/files/13725239/m5x7_16_outline.fnt.txt)
![m5x7_16_outline](https://github.com/korlibs/korge/assets/19785115/f1654133-bf5b-4848-b8c5-f494cdf38b30)

**Comment by soywiz** [comment]
Going to support it, but have in mind that the size of the font is reported to be negative (and that causes like if you use scale=-1):

```
info face="m5x7" size=-16 bold=0 italic=0 charset="" unicode=1 stretchH=100 smooth=0 aa=1 padding=1,1,1,1 spacing=1,1 outline=1
```

You can circumvent it for now by changing size=-16

Couldn't find anything about negative sizes in the spec: https://www.angelcode.com/products/bmfont/doc/file_format.html

**Comment by jobe-m** [comment]
Aaah, yes. Changing `size=-16` to `size=16` does the trick. I am sorry that I did not see that obviously. I guess my fnt files were not valid. I took them from LittleKt. Maybe there was a reason for changing that value in the scope of LittleKt.

I did a quick search for other bitmap fonts and there the size is always a positive number.

I guess we can close this issue.
And taking the absolute value might help in the future to overcome this :)
